zkevm-circuits/src/witness/rlp_fsm.rs

```diff
@@ -772,3 +828,17 @@ pub(crate) struct SmState<F: Field> {
     pub(crate) tag_value_acc: Value<F>,
     pub(crate) tag_bytes_rlc: Value<F>,
 }
+
+#[derive(Clone)]
+pub(crate) struct RlpStackOp<F: Field> {
+    pub is_write: bool,
+    pub id: Value<F>,
+    pub depth: usize,
+    pub value: usize,
+    pub value_prev: usize,
+    pub stack_acc: Value<F>,
+    pub is_init: bool,
+    pub is_push: bool,
+    pub is_pop: bool,
+    pub is_update: bool,
+}
```

Comment:
The only field missing (as compared to the `RlpDecodingTable`) is the `stack_acc_pow_of_rand` field. I suggest we use simply `RlpStackOp` (with that additional field) and remove the `RlpDecodingTable` completely? Wdyt, it removes some LoC

Reply:
`RlpDecodingTable` is removed.